supportconfig/plugins/suse_public_cloud

```diff
@@ -174,14 +174,12 @@ function metadata() {
     if [ -e "/usr/bin/azuremetadata" ]; then
         echo "# /usr/bin/azuremetadata --api latest" >> $INFO_FILE
         /usr/bin/azuremetadata --api latest >> $INFO_FILE
-	remove=("subscriptionId" "subscriptions")
-	_sanitize_data "$INFO_FILE" "${remove[@]}"
     fi
     if [ -e "/usr/bin/gcemetadata" ]; then
         echo "# /usr/bin/gcemetadata" >> $INFO_FILE
         /usr/bin/gcemetadata  >> $INFO_FILE
-	remove=("identity" "token")
```

Comment:
While the identity token is not particularly private it is also somewhat useless data since the default query does not specify an identity. There is principally nothing wrong with keeping it in the supportconfig. On the other hand keeping data that is basically useless in the supportconfig is also questionable.